packages/components/MIGRATING.md

```diff
@@ -66,6 +69,108 @@ Changes:
 </Label>
 ```
 
+### Dialog
+
+Changes:
+
+-   The `Dialog` component now controls its open state internally by default.
+
+    -   You can pass in a trigger component as a child of `Dialog.Trigger` to control the open state.
+
+    -   The `open` and `onOpenChange` props can still be used to control the open state externally if needed.
```

Comment:
Can you add that `handleClose` is now `onOpenChange`

Reply:
added